_code/InSituTree/R/runInSituTree.R

```diff
@@ -0,0 +1,158 @@
+#' Tree-style cell typing
+#'
+#' @param full_profiles Cell profile matrix, genes x cell types, for all end point cell types
+#' @param cth Cell type hierarchy, as a nested list
+#' @param x Counts matrix, cells x genes
+#' @param neg Vector of mean negative controls for each cell
+#' @param name_for_new_annotation Name for this annotation
```

Comment:
I would give a default value for the root level of annotation name.

Reply:
Added.